C/main_libFuzzer_extra_counters.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+
+typedef long long GoInt64;
+typedef GoInt64 GoInt;
+typedef struct { void *data; GoInt len; GoInt cap; } GoSlice;
```

Comment:
Let's pass data pointer/size to Go code instead and reconstruct the slice in Go code using https://golang.org/pkg/reflect/#SliceHeader. The less C code we have, the better.

Reply:
Addressed in latest commit.